core/commands/swarm.go

```diff
@@ -481,6 +408,70 @@ Changes made via command line are persisted in the Swarm.ResourceMgr.Limits fiel
 	},
 }
 
+var swarmResourcesCmd = &cmds.Command{
+	Status: cmds.Experimental,
+	Helptext: cmds.HelpText{
+		Tagline: "Get a summary about all resources in use by libp2p Resource Manager.",
+		LongDescription: `Get a summary about all resources in use by libp2p Resource Manager.
+The output of this command is JSON.
+`},
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		node, err := cmdenv.GetNode(env)
+		if err != nil {
+			return err
+		}
+
+		if node.ResourceManager == nil {
+			return libp2p.ErrNoResourceMgr
+		}
+
+		cfg, err := node.Repo.Config()
+		if err != nil {
+			return err
+		}
+
+		userRessourceOverrides, err := node.Repo.UserRessourceOverrides()
+		if err != nil {
+			return err
+		}
+
+		defaultConfig, _, err := libp2p.LimitConfig(cfg.Swarm, userRessourceOverrides)
+		if err != nil {
+			return err
+		}
+
+		rapi, ok := node.ResourceManager.(rcmgr.ResourceManagerState)
+		if !ok { // NullResourceManager
+			return libp2p.ErrNoResourceMgr
+		}
+
+		stats := rapi.Stat()
+		statsConfig := libp2p.StatToLimitConfig(stats)
+
+		return cmds.EmitOnce(res, libp2p.LimitConfigsToInfo(defaultConfig, statsConfig))
+	},
+	Encoders: cmds.EncoderMap{
+		cmds.Text: cmds.MakeTypedEncoder(func(req *cmds.Request, w io.Writer, ris libp2p.ResourceInfos) error {
+			tw := tabwriter.NewWriter(w, 30, 8, 0, '\t', 0)
+			defer tw.Flush()
+
+			fmt.Fprintf(tw, "%s\t%s\t%s\t%s\t%s\t\n", "scope name", "limit name", "limit", "usage", "percent usage")
```

Comment:
Other commands have these capitalized, perhaps they should be here for consistency?  Also "scope name" can just be "Scope", I don't think "name" suffix adds anything (it does for limit, to distinguish between the name and value...other option is "Limit" and "Limit Value").

Reply:
Also the limit, usage, and percent usage columns are wonky and unintuitive.  Here's an example of me running this on a vanilla kubo daemon:

```
scope name                                                      limit name                      limit                           usage                           percent usage
system                                                          Streams                         -1                              104                             -10400.0%
system                                                          StreamsInbound                  -1                              5                               -500.0%
system                                                          StreamsOutbound                 -1                              99                              -9900.0%
system                                                          Conns                           -1                              111                             -11100.0%
system                                                          ConnsInbound                    4100                            -2                              -0.0%
system                                                          ConnsOutbound                   -1                              111                             -11100.0%
system                                                          FD                              2524288                         15                              0.0%
system                                                          Memory                          4300000000                      3670016                         0.1%
transient                                                       Streams                         -1                              1                               -100.0%
```
These negative percents are bogus, they are computing a value against an enum value which is nonsensical. And I don't know why usage is "BlockAllLimit"? Can we show the string value instead of the numeric value for the limit and usage (e.g. "unlimited" instead of -1) and then show n/a or something for the % usage if the limit is not Unlimited nor BlockAllLimit ?